app/views/my_modules/_module_header.html.erb

```diff
@@ -11,7 +11,7 @@
 
   <div class="col-xs-6 col-sm-6 col-md-4">
     <div class="badge-icon bg-primary">
-      <% if can_edit_module(@my_module) then %>
+      <% if can_manage_experiment?(@my_module.experiment) then %>
```

Comment:
remote  "then" is useless here

Reply:
I agree, but it's also used elsewhere. This should be left for refactoring.